### PR TITLE
Products listing

### PR DESCRIPTION
**Pull request by aatishpandey**
added styles and structure for product listing page
image

### CODE REVIEW

File: css/pages/products-list.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    grid-template-columns: 25rem auto;
+    grid-template-areas:
+    "pro-side prod-cont prod-cont prod-cont";
+
+}
+
+.product-sidebar{
+    grid-area : pro-side;
+    height: 100%;
+    background-color: beige;
```
Comment (srejitk) [inline css/pages/products-list.css]: Would suggest to use color Variables here.

File: pages/products.html [hunk 2]
```diff
@@ -0,0 +1,411 @@
...
+                    <input id="4star" type="radio">
+                    Price high to Low
+                </label>
+             </div>
+
+        </div>
+
+        <!-- product container -->
+        <div class="product-container">
+            <div class="sidebar-head">Showing All Products <i class="sidebar-text">(Showing 100 products)</i>
```
Comment (srejitk) [inline pages/products.html:141]: Use Semantic elements for headers.

**Review by nakulsharma15 — APPROVED** [review]
Looks good to me. No changes suggested!

**Review by srejitk — APPROVED** [review]
Suggested some minor changes. Code looks good otherwise.

**Pull request merged**